frontend/packages/noobaa-storage-plugin/src/components/data-consumption-card/data-consumption-card-dropdown.tsx

```diff
@@ -0,0 +1,116 @@
+import * as React from 'react';
+import { Dropdown, DropdownToggle, DropdownItem, DropdownPosition } from '@patternfly/react-core';
```

Comment:
you should use `Dropdown` from `utils/dropdown.jsx`

Reply:
@rwagner that dropdown dont have the functionality to disable dropdown items. Thats why I used PF4!
![dropdoe](https://user-images.githubusercontent.com/25664409/61357156-6800ff00-a895-11e9-8a3c-2d2a8b1f43b8.png)